### PR TITLE
81 add new items

### PR DESCRIPTION
**Pull request by MLHumphries**
- Enemies attacks and stats will still need adjusting
- Player attacks will need creating. With that, do we want each of us to create our own version of ourselves?

### CODE REVIEW

File: dunwoody_disaster/CharacterFactory.py [hunk 1]
```diff
@@ -258,6 +258,16 @@ def Cooper() -> Character:
...
+        # char.weapons = [
+        #     Item.Weapon(name="Keyboard", damage=5, magic=5, stamina=5, magical=True),
+        #     Item.Weapon(name="", damage=5, magic=5, stamina=5, magical=False),
+        #     Item.Weapon(name="", damage=5, magic=5, stamina=5, magical=False)
+        # ]
+
+        # char.defenses = [
+        #     Item.Armor(name="", armorVal=5, staminaCost=5, magicCost=5, magicDefense=0),
+        #     Item.Armor(name="", armorVal=5, staminaCost=5, magicCost=5, magicDefense=5)
+        # ]
```
Comment (NoahBartell) [inline dunwoody_disaster/CharacterFactory.py:273]: If we aren't using it we can delete it

File: dunwoody_disaster/CharacterFactory.py [hunk 2]
```diff
@@ -278,12 +288,23 @@ def John() -> Character:
...
+        # char.weapons = [
+        #     Item.Weapon(name="", damage=5, magic=5, stamina=5, magical=True),
+        #     Item.Weapon(name="", damage=5, magic=5, stamina=5, magical=False),
+        #     Item.Weapon(name="", damage=5, magic=5, stamina=5, magical=False)
+        # ]
+
+        # char.defenses = [
+        #     Item.Armor(name="", armorVal=5, staminaCost=5, magicCost=5, magicDefense=0),
+        #     Item.Armor(name="", armorVal=5, staminaCost=5, magicCost=5, magicDefense=5)
+        # ]
```
Comment (NoahBartell) [inline dunwoody_disaster/CharacterFactory.py:304]: Same here

File: dunwoody_disaster/CharacterFactory.py [hunk 3]
```diff
@@ -294,12 +315,23 @@ def Noah() -> Character:
...
```
Comment (NoahBartell) [inline dunwoody_disaster/CharacterFactory.py:331]: and here

File: dunwoody_disaster/CharacterFactory.py [hunk 4]
```diff
@@ -310,6 +342,17 @@ def Mitch() -> Character:
...
+        # char.weapons = [
+        #     Item.Weapon(name="", damage=8, magic=15, stamina=8, magical=True),
+        #     Item.Weapon(name="", damage=10, magic=5, stamina=10, magical=False),
+        #     Item.Weapon(name="", damage=15, magic=5, stamina=15, magical=False)
+        # ]
+
+        # char.defenses = [
+        #     Item.Armor(name="Stress Ball", armorVal=1, staminaCost=10, magicCost=0, magicDefense=15),
+        #     Item.Armor(name="Spaghetti Code", armorVal=5, staminaCost=5, magicCost=5, magicDefense=5)
+        # ]
```
Comment (NoahBartell) [inline dunwoody_disaster/CharacterFactory.py:358]: ..and here

**Review by NoahBartell — APPROVED** [review]
Besides the clean up everything looks good to me. Nice job!

**Comment by NoahBartell** [comment]
Make sure to update with main

File: dunwoody_disaster/CharacterFactory.py (hunk 1 above)
Comment (chanson02) [reply dunwoody_disaster/CharacterFactory.py:273]: We will use it, there's just no values to put there yet.

**Comment by chanson02** [comment]
Meant to put [this](https://github.com/chanson02/DunwoodyDisaster/pull/62#issuecomment-2093754858) here. Too many tabs lol.

**Review by instMeyFTW — APPROVED** [review]
Great work

**Comment by NoahBartell** [comment]
@MLHumphries you can make sure the branch is up to date and merge

**Pull request merged**